### ISSUE TITLE
[BUG] Web component issues with WCAG20_Input_RadioChkInFieldSet

### ISSUE THREAD

**Issue by tombrunet** [IBMa/equal-access#381]
Example: https://web-components.carbondesignsystem.com/vue/iframe.html?id=components-radio-button--default&viewMode=story

While the example doesn't have a parent radiogroup, if it's added, the rule doesn't properly handle it. Related to #363

**Comment by tombrunet** [comment]
Fixed on preview archive.
